### DISCUSSION TITLE
How to keep a static splash screen theme in the mobile app?

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-Ed/Community#337] [General Discussion]
Scenario:
I have a dummy app, I want to have the same splash screen whenever I open the app. But currently there are 2 types of splash screen showing:

1. When app is opened for the first time, able to see Lorem Ipsum icon with white background
2. When app is closed and opened again, able to see Lorem Ipsum icon with a theme background and a title **Welcome to Lorem Ipsum**

How can we customize the theme or use same theme in both 1 and 2 ?

Sharing the video as well:

https://user-images.githubusercontent.com/107101132/210523641-0a1df519-4364-4c2c-b10e-202c31d28ce9.mp4

to: @reshmi-nair @rajeevsathish @Krishnaj20 @swayangjit
cc: @baljeetsingh

**Comment by swayangjit** [comment]
@nidhikachhawa24 So you want the yellow theme to get applied by default right?

**Reply by nidhikachhawa24** [reply]
@swayangjit Yes

**Reply by swayangjit** [reply]
@nidhikachhawa24 I will check how we can use the theme and update in this thread.

**Reply by swayangjit** [reply]
@nidhikachhawa24 By default we are enabling the "DEFAULT" theme . You can change it to "JOYFUL" theme you can set the default theme to "JOYFUL" in the following line
https://github.com/project-sunbird/cordova-plugin-sunbirdsplash/blob/4d4aaeb3029a3e3c684e305780dac463b329c27c/src/android/org/sunbird/SplashScreen.java#L236

**Reply by Krishnaj20** [reply]
@nidhikachhawa24 - Pls mark this thread as answered, if your query is resolved.
